tmt/base.py

```diff
@@ -4658,6 +4660,21 @@ def show_runner(self, logger: tmt.log.Logger) -> None:
         for _, key_formatted, value_formatted in self.runner.facts.format():
             log(key_formatted, value_formatted, shift=1)
 
+    def copy_scripts(self) -> None:
+        """
+        Copy the tmt helper scripts under the running workdir
+        into a new scripts directory.
+        """
+        assert self.workdir is not None
+        scripts = tmt.steps.scripts.SCRIPTS
+        destination = self.workdir / "scripts"
+        destination.mkdir(exist_ok=True)
+
+        for script in scripts:
```

Comment:
```suggestion
        for script in tmt.steps.scripts.SCRIPTS:
```

The variable is used just once, no need for the extra line.